src/tee/snp.rs

```diff
@@ -0,0 +1,12 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub struct SnpRequest {
+    pub workload_id: String,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct SnpAttestation {
+    pub nonce: String,
+    pub measurement: String,
```

Comment:
It seems like this should be the entire Attestation Report, rather than just a measurement. Also, do you want the cert chain in here as well?

Reply:
Yes, I agree. Sorry, this is a WIP and I haven't been pushing my latest changes as of late. Will do that within the next week. I've included the entire report and cert chain.